### DISCUSSION TITLE
Nice New Logo

### DISCUSSION THREAD

**Discussion by crispybegs** [rommapp/romm#175] [General]
Good work on the logo @zurdi15 . Looks great in my homepage dashboard.

<img width="557" alt="Screenshot 2023-04-14 at 15 38 20" src="https://user-images.githubusercontent.com/129169122/232075237-5bea359c-464a-4949-b37c-d65be5bc0646.png">

**Comment by zurdi15** [comment]
Yay! Thank you mate 😄 
 
I am working in the RomM branding with new logo and new color palette in the next version
